lib/utils.tsx

```diff
@@ -68,3 +69,10 @@ export const sortByOldest = (a: IPin, b: IPin) => {
     DateTime.fromISO(a.date).toMillis() - DateTime.fromISO(b.date).toMillis()
   );
 };
+
+export function getAuthorIcon(author: string | string[]) {
```

Comment:
This should be a component because it it JSX 🙌

Reply:
Done :rocket: